cmd/deploy.go

```diff
@@ -49,9 +49,16 @@ func newDeployCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var deployCmd = &cobra.Command{
 		Use:   "deploy",
-		Short: "Build and deploy your Appsody project to your Kubernetes cluster",
-		Long: `This command extracts the code from your project, builds a local Docker image for deployment,
-generates a deployment manifest (yaml) file if one is not present, and uses it to deploy your image to a Kubernetes cluster, either via the Appsody operator or as a Knative service.`,
+		Short: "Build and deploy your Appsody project to Kubernetes.",
+		Long: `Build and deploy a local container image of your Appsody project to your Kubernetes cluster. 
+		
+The stack, along with your Appsody project, is extracted to a local directory before the container image of your Appsody project is built for deployment.
+The command generates a deployment manifest file, "app-deploy.yaml", if one is not present. The CLI uses this file to deploy your image to your Kubernetes cluster via the Appsody operator, and optionally as a Knative service if you specify the "--knative" flag.`,
```

Comment:
This part is now done as part of `appsody build` - I suppose since deploy does a build under the covers this is still technically correct though. However, does the new help text for `appsody build` reference this?

Reply:
Correct. I think all of this is fine to keep here. Somewhere we may want to say that this runs an `appsody build` first, then applies the `app-deploy.yaml` to the cluster. 

The `appsody build` help text probably needs review.

Reply:
We should probably also add something about the appsody operator. The operator is installed and used for the deployment.

Reply:
@kylegc I have made the requested changes to this command. Do we need to mention more about the Appsody Operator in this command past "The CLI uses this file to deploy your image to your Kubernetes cluster via the Appsody Operator...."?

Reply:
As I said in the other comments, I think we should include a simple statement that says if the operator doesn't exist, it will be installed.